### PR TITLE
Include email reporters on Documentation

### PR DESCRIPTION
**Pull request by verracca**

### CODE REVIEW

File: modules/ROOT/pages/bat-reporting-task.adoc
Comment (rddheath) [inline modules/ROOT/pages/bat-reporting-task.adoc]: What do you mean by "when using the Anypoint Platform"? Using it in what way? 
Do the listed email addresses need to be associated with accounts in Anypoint Platform?

File: modules/ROOT/pages/bat-reporting-task.adoc
Comment (rddheath) [inline modules/ROOT/pages/bat-reporting-task.adoc]: How does this system need to be configured?

Comment (ferferrarazzo) [reply modules/ROOT/pages/bat-reporting-task.adoc]: @rddheath I changed the doc. can you take a look?

**Comment by rddheath** [comment]
I'll have to review this next week, when I can spend more time on it and ask you questions. This week is packed.

**Review by rddheath — APPROVED** [review]
Do not merge this PR, yet. If you do, the changes will go to the Docs team's staging server and will be published today. We need to wait until Friday to merge the PR.

**Pull request merged**
